src/engine/passwordless/social_or_phone_number_login_screen.jsx

```diff
@@ -35,13 +38,21 @@ const Component = ({ i18n, model }) => {
     />
   ) : null;
 
+  const captchaPane =
+  l.captcha(model) &&
+  l.captcha(model).get('required') &&
+  (isHRDDomain(model, databaseUsernameValue(model)) || !sso) ? (
+    <CaptchaPane i18n={i18n} lock={model} onReload={() => swapCaptcha(l.id(model), false)} />
+  ) : null;
```

Comment:
I haven't tested this flow but this will likely have the same runtime problem (it also tries to use `sso`).